tests/test_copy_config.c

```diff
@@ -2209,6 +2209,6 @@ main(void)
     };
 
     setenv("CMOCKA_TEST_ABORT", "1", 1);
-    sr_log_stderr(SR_LL_INF);
```

Comment:
Where did this call disappear? It is not part of `test_log_init()`.

Reply:
sr_log_stderr call is not necessary, as we will be logging due to log_cb and not fprintf.

```c
/* stderr logging */
    if (ll <= stderr_ll) {
        fprintf(stderr, "[%s] %s\n", severity, msg);
    }

    /* syslog logging */
    if (ll <= syslog_ll) {
        syslog(priority | (plugin ? LOG_DAEMON : 0), "[%s] %s\n", severity, msg);
    }

    /* logging callback */
    if (log_cb) {
        log_cb(ll, msg);
    }
```